src/components/input/code/views/inputView.jsx

```diff
@@ -46,7 +47,7 @@ module.exports = React.createClass({
     var isValid = true;
     var error = null;
 
-    if(value !== '' && typeof self.props.validator !== 'undefined'){
```

Comment:
I'm going to revert back to this line, but I think hard type checking `value` against a `String` is probably not a great assumption for `input` fields.